lib/plugin.js

```diff
@@ -1,30 +1,26 @@
 const cds = require("@sap/cds");
-const { Logger } = require("./logger");
-const { ord, getMetadata, defaults } = require("./");
+const { ord, getMetadata, defaults, Logger } = require("./");
+const { authenticate } = require("./authentication");
 
-
-cds.on("bootstrap", (app) => {
+cds.on("bootstrap", async (app) => {
     app.use("/.well-known/open-resource-discovery", async (req, res) => {
-        if (req.url === "/") {
-            res.status(200).send(defaults.baseTemplate);
-        } else {
-            try {
-                const { contentType, response } = await getMetadata(req.url);
-                res.status(200).contentType(contentType).send(response);
-            } catch (error) {
-                Logger.error(error, 'Error while generating metadata');
-                res.status(500).send(error.message);
-            }
-        }
+        res.status(200).send(defaults.baseTemplate);
     });
 
-    app.get("/open-resource-discovery/v1/documents/1", async (req, res) => {
+    app.get("/ord/v1/documents/*", authenticate, async (req, res) => {
```

Comment:
I didn't see the change to the ord config baseTemplate reflecting this endpoint change.